spec/helper.rb

```diff
@@ -7,6 +7,7 @@
 require 'silverpop'
 require 'rspec'
 require 'webmock/rspec'
+require 'pry'
```

Comment:
It has to be required somewhere and I was using it to debug tests.  I can remove it from here if you'd like.  It wouldn't make sense to require it in any of the actual gem files.